### PR TITLE
fix: loading target to all pages after 3 seconds

### PR DESCRIPTION
**Pull request by DanielHe4rt**
## Motivation

After the release of 0.0.3 we saw that the plugin was only loaded in one page per browser due each page loading time.

Currently the goal is to support:
- Streamer Dashboard - dashboard.twitch.tv/*
- Twitch Streamer Page - twitch.tv/*
- Embed Chats - twitch.tv/embed/*

## Fixes

I've tried to add EventListeners on window but still not working properly. After that I just decided to implement a small recursion to start to listen when the page is fully loaded.

- Added setInterval(() => {}, 3000 for the targeted pages.

### CODE REVIEW

File: extension/src/content.ts [hunk 1]
```diff
@@ -1,3 +1,24 @@
+{
+}
+
```
Comment (Daniel-Boll) [inline extension/src/content.ts:3]: This is odd, I think it can be removed.

**Review by Daniel-Boll — APPROVED** [review]
LGTM

File: extension/src/content.ts (hunk 1 above)
Comment (DanielHe4rt) [reply extension/src/content.ts]: You can't remove it. If you do weird stuff will start to happen (Plasmo Docs etc).

Comment (Daniel-Boll) [reply extension/src/content.ts]: LOL

**Pull request merged**